<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogImageProperty</class>
 <widget class="QDialog" name="DialogImageProperty">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Size</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="2">
         <widget class="QComboBox" name="comboWidthUnit">
          <item>
           <property name="text">
            <string>pixels</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>%</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinWidth">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>99999</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkWidth">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Width</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinHeight">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>99999</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QComboBox" name="comboHeightUnit">
          <item>
           <property name="text">
            <string>pixels</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>%</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkHeight">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Height</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushEdit">
     <property name="text">
      <string>Edit image</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Image address</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="comboAction">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Create link to original place</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Copy image to doc folder</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Move image to doc folder</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushCreate">
       <property name="text">
        <string>Create image</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="lineFPath"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushOpen">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>178</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogImageProperty</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogImageProperty</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
